tests/integration/test_headers_and_subtotals.py

```diff
@@ -2621,115 +2621,112 @@ def it_calculates_residuals_for_cat_x_items_x_cats_with_col_insertion(self):
-                    0.00153552,
-                    0.00212325,
-                    0.00323218,
-                ],
-                [
-                    0.0006797,
-                    0.00049171,
-                    0.00056298,
-                    0.00085127,
-                    0.00134268,
-                    0.00159858,
-                    0.00232773,
-                    0.00326122,
-                    0.00282484,
-                    0.00350006,
-                    0.00500803,
-                ],
-                [
-                    0.00051466,
-                    0.00040214,
-                    0.0004936,
-                    0.00065933,
-                    0.00110404,
-                    0.0013077,
-                    0.00184918,
-                    0.0023456,
-                    0.00193395,
-                    0.00257512,
-                    0.0038076,
-                ],
-                [
-                    0.00022847,
-                    0.00025662,
-                    0.00020738,
-                    0.00021275,
-                    0.00065742,
-                    0.00068634,
-                    0.00106073,
-                    0.00118541,
-                    0.00104104,
-                    0.00129328,
-                    0.00201798,
-                ],
-                [
-                    0.00046588,
-                    0.00026735,
-                    0.00024763,
-                    0.00027782,
-                    0.00079212,
-                    0.0010403,
-                    0.0015337,
-                    0.00188601,
-                    0.00185383,
-                    0.00233187,
-                    0.00340655,
-                ],
-                [
-                    0.00040699,
-                    0.00016913,
-                    0.00023541,
-                    0.0003879,
-                    0.00061709,
-                    0.00084293,
-                    0.00116563,
-                    0.00153654,
-                    0.00147585,
-                    0.00165358,
-                    0.00264447,
-                ],
-                [
-                    0.00027773,
-                    0.00016999,
-                    0.0001759,
-                    0.00029045,
-                    0.00055084,
-                    0.00054703,
-                    0.0007302,
-                    0.00096937,
-                    0.00071786,
-                    0.00093394,
-                    0.00151657,
-                ],
-                [
-                    0.00022371,
-                    0.00017666,
-                    0.0002176,
-                    0.0002579,
-                    0.00052932,
-                    0.00047783,
-                    0.00082904,
-                    0.00105215,
-                    0.00090766,
-                    0.00107047,
-                    0.00174012,
-                ],
+        assert slice_.table_std_err.tolist() == [
```

Comment:
why thid expectation has changed? and more important, why this has a different shape than the old one?

Reply:
I don't think it has a new shape? I count it as 8 X 11 in both.

It changed because in the legacy matrix the bases for cat arrays sums across the cat array subvariables but in the new new ones they don't. Therefore, the `table_proportions` are different, and thus the `table_variance` changes.

Reply:
sorry my bad :(